components/ironic/values.yaml

```diff
@@ -88,8 +88,16 @@ conf:
       loader_file_paths: "snponly.efi:/usr/lib/ipxe/snponly.efi"
     inspector:
       extra_kernel_params: ipa-collect-lldp=1
+      # Agent inspection hooks - ports hook removed to prevent port manipulation during agent inspection
+      # Default hooks include: ramdisk-error,validate-interfaces,ports,architecture
+      # We override to exclude 'ports' from the default hooks
+      default_hooks: "ramdisk-error,validate-interfaces,architecture"
       hooks: "$default_hooks,pci-devices,parse-lldp,local-link-connection,resource-class"
     # enable sensors and metrics for redfish metrics - https://docs.openstack.org/ironic/latest/admin/drivers/redfish/metrics.html
+    redfish:
+      # Redfish inspection hooks - keep ports hook to allow port manipulation during redfish inspection
+      # Default hooks include: validate-interfaces,ports,architecture
+      inspection_hooks: "$default_inspection_hooks,pci-devices,parse-lldp,local-link-connection,resource-class"
```

Comment:
This is a new block. I would drop this because its adding extra hooks for redfish.